.github/workflows/create-cluster.yml

```diff
@@ -58,8 +58,7 @@ on:
 
 env:
   ACCEPT_RAW: "Accept: application/vnd.github.v3.raw"
-  script_url: /repos/${{github.repository}}/contents/.github/workflows/scripts/common.sh?ref=${{github.event.repository.default_branch}}
-  main_branch: ${{github.event.repository.default_branch}}
```

Comment:
Removing `main_branch` here as I didn't see any other usage besides in `common.sh` in this workflow.